scripts/eks_existing_vpc_locals.snip

```diff
@@ -1,5 +1,4 @@
   vpc_id                 = "{{ .VPCID }}"
-  private_route_table_id = "{{ .PrivateRouteTableID }}"
```

Comment:
Removed this since it is no more needed. The example does an EKS Agentless deployment using a public cluster for which peering is not required anymore.